sources/engine/Stride.Assets/Stride.Assets.csproj

```diff
@@ -65,25 +65,6 @@
       <Link>Properties\SharedAssemblyInfo.cs</Link>
     </Compile>
   </ItemGroup>
-  <ItemGroup>
```

Comment:
Removing those `<None Include` because it references files under Stride.Engine and Stride.UI - blame shows this is a remnant from the initial commit.
I'm guessing those were added to allow MSBuild to track changes in assets files and rebuild the project when one of them changes. Currently that is not in place elsewhere and changing assets requires manually touching any C# related files to trigger MSBuild to rebuild the project.